runtime/include/python/chplrt.pxd

```diff
@@ -27,10 +27,16 @@ cdef extern from "chpl-external-array.h":
 	void cleanupOpaqueArray(chpl_opaque_array* arr)
 
 cdef extern from "chpl-export-wrappers.h":
-	ctypedef struct chpl_bytes_wrapper:
+	ctypedef struct chpl_byte_buffer:
 		int isOwned
 		char* data
 		size_t size
 
-	void chpl_bytes_wrapper_free(chpl_bytes_wrapper cb)
+	void chpl_byte_buffer_free(chpl_byte_buffer cb)
+
+cdef extern from "chpl-mem.h":
+	void chpl_free(void* ptr)
+
+cdef extern from "string.h":
```

Comment:
Is there not a way to import this from something Cython provides, since it is a common C function as opposed to something we provide?

Reply:
Now that I think about it, I believe I can go back to the original code for `c_string`. This was an artifact on this branch from before I mapped the Chapel string to a byte buffer.

Thanks!